### PR TITLE
[RW-6249][risk=low] Update default repo branch name from master -> main

### PR DESCRIPTION
**Pull request by calbach**
The plan would be to rename the default branch from master -> main, then immediately merge this PR. Probably something unexpected will break as well, and we'll fix that forward.

* Primarily: update any Circle, deploy, or devops tooling which would break as a result of the rename
* Also included: updates for README's / broken links

### CODE REVIEW

File: api/libproject/devstart.rb [hunk 1]
```diff
@@ -1058,7 +1058,7 @@ def circle_build_cdr_indices(cmd_name, args)
   op.add_option(
     "--branch [--branch]",
     ->(opts, v) { opts.branch = v},
-    "Branch. Optional - Default is master."
+    "Branch. Optional - Default is main."
```
Comment (jmthibault79) [inline api/libproject/devstart.rb:1061]: interesting - the default was already updated here

File: hooks/pre-commit [hunk 2]
```diff
@@ -7,9 +7,9 @@ eval "$(git rev-parse --show-toplevel)/hooks/apply-git-secrets.sh"
 git secrets --pre_commit_hook -- "$@"
 
 CURRENT_BRANCH=`git rev-parse --abbrev-ref HEAD`
-if [ "$CURRENT_BRANCH" = "master" ]
+if [ "$CURRENT_BRANCH" = "main" ]
```
Comment (jmthibault79) [inline hooks/pre-commit:10]: I'm sure there will be many local `master` branches for a while, so let's keep both for now.  Maybe we could have a different error like "the default branch is now main"

**Review by jmthibault79 — APPROVED** [review]
I search for any remaining "master"s in the repo and I didn't find any.

File: api/libproject/devstart.rb (hunk 1 above)
Comment (calbach) [reply api/libproject/devstart.rb:1061]: I believe this one is referring to workbench-devops, which I moved over a while back (and apparently missed this doc update)

File: hooks/pre-commit (hunk 2 above)
Comment (calbach) [reply hooks/pre-commit:10]: Done

**Pull request merged**